test/server/configuration-test.js

```diff
@@ -274,10 +293,29 @@ describe('TestCafeConfiguration', function () {
                         });
                 });
             });
+
+            it('Should warn message on multiple configuration files', async () => {
+                createJsTestCafeConfigurationFile({
+                    'hostname': '123.456.789',
+                    'port1':    1234,
+                    'port2':    5678,
+                    'src':      'path1/folder',
+                    'browser':  'ie'
+                });
+
+                consoleWrapper.wrap();
+                await testCafeConfiguration.init();
+                consoleWrapper.unwrap();
+                await del([testCafeConfiguration.defaultPaths[jsFilePriority]]);
+
+                const configPriorityListStr = testCafeConfiguration.defaultPaths.map((path, index) => `${index + 1}. ${path}`).join('\n') || '';
+
+                expect(consoleWrapper.messages.log).eql(renderTemplate(WARNING_MESSAGES.multipleConfigurationFilesFound, testCafeConfiguration.filePath, configPriorityListStr));
```

Comment:
Compare with plain string. It allowing to find excess new lines and commas and so on.